javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,35 @@
+getEventWeekday = (daysToEvent) => {
+  const d = new Date();
+  const today = d.getDay();
+
+  const q = (daysToEvent + today) % 7;
+
+  for (i = 1; i <= 7; i++) {
+    switch (q) {
```

Comment:
Great name with the daysToEvent and today 👍🏾, I'll suggest you change the q to something very descriptive as well.

Reply:
committed changes